### DISCUSSION TITLE
Newbie questions - dist folder

### DISCUSSION THREAD

**Discussion by ccfai8** [difurious/lightweight-charts-line-tools#6] [Q&A]
After generating the dist folder, should we place the node_modules lightweight chart folder documents?

I think we should regard it as a standalone plugin but when I import it as module, I see errors:

File '/public/vendor/dist/lightweight-charts.standalone.development.js' is not a module.​​​​​​​​​​​​​​​​

I also see
Property 'addLineTool' does not exist on type 'IChartApi'.

Hopefully can get some help here. Much appreciated!

**Comment by difurious** [comment]
@ccfai8  so you were able to compile it because it spit out the version in the dist folder.  Now you need to import one of the js files.  Look at /debug.html and that is a simple import.

Im using react and node, so I import line-tools with something like this.  Its the root folder that you just built the lint-tools in and not the dist folder. So something like "C:\line_tools_project_root_folder"  not "C:\line_tools_project_root_folder\dist"  
`npm install /PATH_TO_YOUR_LINE_TOOLS_ROOT_FOLDER`

in the debug.html you see an example of making a chart, then a series, then an example on how to use the .addLineTool for each tool, so use that as reference.

I hope that you understand that you need to do a lot of frontend work to make it look like the examples and its not out of the box going to look like Trading View just because you compiled and imported the line-tools-build.

**Reply by ccfai8** [reply]
Thanks [difurious](https://github.com/difurious)! I installed it successfully but then I see many problems/errors popping up when i open a few documents. I have to think about the next step... thanks again!
